consensus/processor.go

```diff
@@ -268,14 +279,21 @@ func (p *DefaultProcessor) ProcessTipSet(ctx context.Context, st state.Tree, vms
 //       revert errors.
 //   - everything else: successfully applied (include, keep changes)
 //
-func (p *DefaultProcessor) ApplyMessage(ctx context.Context, st state.Tree, vms vm.StorageMap, msg *types.SignedMessage, minerOwnerAddr address.Address, bh *types.BlockHeight, gasTracker *vm.GasTracker, ancestors []types.TipSet) (*ApplicationResult, error) {
+func (p *DefaultProcessor) ApplyMessage(ctx context.Context, st state.Tree,
```

Comment:
nit: when moving from horizontal to vertical I like to commit all the way and put one argument per line:
```
ApplyMessage(
  ctx context.Context,
  st state.Tree,
...
  ancestors []types.TipSet,
)
```